src/Place/ReadModel/RDF/RdfProjector.php

```diff
@@ -112,7 +116,10 @@ private function setGeneralProperties(Graph $graph, string $uri, DomainMessage $
 
         // Add/update the generiek:versieIdentificator inside the linked adms:Identifier on every change.
         $identificator = $resource->getResource(self::PROPERTY_LOCATIE_IDENTIFICATOR);
-        $identificator->set(self::PROPERTY_IDENTIFICATOR_VERSIE_ID, $recordedOnLiteral);
+        $identificator->set(
+            self::PROPERTY_IDENTIFICATOR_VERSIE_ID,
+            new Literal($recordedOn->format(DateTime::ATOM), null, 'xsd:string')
```

Comment:
`$recordedOn->format(DateTime::ATOM)`

Is used 3 times, maybe we could move it to a variable?

Reply:
Done! :)

Reply:
✅